README.md

```diff
@@ -67,6 +67,23 @@ OutlineView(data, children: \.children, selection: $selection) { item in
 
 ### Customization
 
+#### Children
+There are two types of `.children` parameters in the `OutlineView` initializers. You can either provide children for data items using a KeyPath pointing to an optional array of the same type, or a closure that provides an optional array based on the `ID` of the item.
```

Comment:
```suggestion
There are two types of `children` parameters in the `OutlineView` initializers. You can either provide the children for an item using a `KeyPath` pointing to an optional `Sequence` of the same type, or a closure that provides an optional `Sequence` based on the `ID` of the item.
```

Reply:
Got it https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3